pack/23s.json

```diff
@@ -110,9 +110,9 @@
         "quantity": 3,
         "side_code": "runner",
         "strength": 1,
-        "stripped_text": "2 credits: Break all subroutines. Use this ability only by spending credits from stealth cards. 1 credit: +1 strength.",
+        "stripped_text": "Interface -> 2click: Break all subroutines. Use this ability only by spending credits from stealth cards. 1click: +1 strength.",
```

Comment:
Shouldn't this (and the other costs for Dai V be `[credit]`?

Reply:
Looks like a bug from Nisei. I'll let them know and fix it here.